ext/opentelemetry-ext-testutil/src/opentelemetry/ext/testutil/wsgitestutil.py

```diff
@@ -1,32 +1,28 @@
 import io
 import unittest
-import unittest.mock as mock
 import wsgiref.util as wsgiref_util
 
 from opentelemetry import trace as trace_api
+from opentelemetry.sdk.trace import TracerSource, export
+from opentelemetry.sdk.trace.export.in_memory_span_exporter import (
+    InMemorySpanExporter,
+)
 
 
 class WsgiTestBase(unittest.TestCase):
-    def setUp(self):
-        self.span = mock.create_autospec(trace_api.Span, spec_set=True)
-        tracer = trace_api.Tracer()
-        self.get_tracer_patcher = mock.patch.object(
-            trace_api.TracerSource,
-            "get_tracer",
-            autospec=True,
-            spec_set=True,
-            return_value=tracer,
-        )
-        self.get_tracer_patcher.start()
-
-        self.start_span_patcher = mock.patch.object(
-            tracer,
-            "start_span",
-            autospec=True,
-            spec_set=True,
-            return_value=self.span,
+    @classmethod
+    def setUpClass(cls):
```

Comment:
We'll need to add a `tearDownClass` that undoes this (probably using `importlib.reload`). Otherwise, as soon as we have a second unit test that uses the same pattern, we'll get problems. See also the loader tests in the API package.

Reply:
Thanks for the pointer, I did not know about the reload method, will updated the test.

Reply:
Updated